realm/realm-library/src/androidTest/java/io/realm/NotificationsTest.java

```diff
@@ -425,91 +443,6 @@ public void onChange(Realm object) {
 
     @Test
     @RunTestInLooperThread
-    public void weakReferenceListener() throws InterruptedException {
```

Comment:
Any reason to remove these? Listener reachability doesn't seem connected to the `Results` class?